codec/src/main/java/io/netty/handler/codec/compression/StandardCompressionOptions.java

```diff
@@ -0,0 +1,71 @@
+/*
+ * Copyright 2021 The Netty Project
+ *
+ * The Netty Project licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *   https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+package io.netty.handler.codec.compression;
+
+import com.aayushatharva.brotli4j.encoder.Encoder;
+
+/**
+ * Standard Compression Options for {@link BrotliOptions},
+ * {@link GzipOptions} and {@link DeflateOptions}
+ */
+public final class StandardCompressionOptions {
+
+    private StandardCompressionOptions() {
+        // Prevent outside initialization
+    }
+
+    /**
+     * @see BrotliOptions#DEFAULT
```

Comment:
This, as well as other member references in this class's Javadoc, won't linked properly in generated Javadoc because they are not public. We need to write a proper first sentence, such as:

```
 * Returns the {@link BrotliOptions} with the default compression settings.
```

Reply:
@trustin want to do a PR ?

Reply:
I will do it as I've upcoming PR to enable Brotli tests on Windows.